index.html

```diff
@@ -26,273 +26,309 @@
     <script src="index.js">
         
     </script>
+    <!--Awesome Fonts!-->
+    
     <title>Fabrizio's Portfolio</title>
 </head>
-<body>
-    <header>
-        <nav class="toolbar">
-            <div class="desktop-toolbar">
-                <a href="#recentWorks">Portfolio</a>
-                <a href="#aboutme">About</a>
-                <a href="#contactme">Contact</a>
-            </div>
+<body onload="assignEventHandlers()">
+    <main>
+        <header>
+            <nav class="toolbar">
+                <div class="desktop-toolbar">
+                    <a href="#recentWorks">Portfolio</a>
+                    <a href="#aboutme">About</a>
+                    <a href="#contactme">Contact</a>
+                </div>
 
 
-            <a href="#" id="welcomeText">
-                Welcome
-            </a>
-            <a class="menuButton" onclick="showMenu()">
-                <img src="src/header/headerButton.svg" alt="">
-            </a>
+                <a href="#" id="welcomeText">
+                    Welcome
+                </a>
+                <a class="menuButton">
+                    <img src="src/header/headerButton.svg" alt="">
+                </a>
 
-            <div class="menu">
-                
-                <div class="menu-content">
-                    <a href="#" class="menu-close" onclick="closeMenu()">
-                        <img src="./src/misc/close-menu.svg" alt="">
+                <div class="menu">
+                    
+                    <div class="menu-content">
+                        <a href="#" class="menu-close" onclick="closeMenu()">
+                            <img src="./src/misc/close-menu.svg" alt="">
```

Comment:
- [ ] OPTIONAL
To improve the accessibility feature of your page kindly endeavor to include `alt` names to your images